contributed_definitions/nyaml/NXsts.yaml

```diff
@@ -818,13 +834,10 @@ NXsts(NXsensor_scan):
     sample(NXsample):
       exists: optional
       doc: |
-        To describe the sample and other constraints that are applied to that sample
-        before scanning.
+        To describe sample and other constaints that applied on sample before scanning.
       sample_prep_descripton:
-        exists: optional
         doc: |
-          At this moment no base class is available that can track entire sample
-          preparation.
+          At this moment no base class available that can track entire sample preparation.
```

Comment:
This is not true, please have a look at `NXhistory` and its use in `NXmpes/NXsample/history`.

Reply:
I am taking this note for final `SPM` app def currently we are woking on.

Reply:
https://github.com/FAIRmat-NFDI/nexus_definitions/pull/241